IdentityCore/src/validation/MSIDB2CAuthority.m

```diff
@@ -165,4 +165,17 @@ + (NSURL *)normalizedAuthorityUrl:(NSURL *)url
     return [NSURL URLWithString:normalizedAuthorityUrl];
 }
 
+#pragma mark - NSCopying
+
+- (id)copyWithZone:(NSZone *)zone
+{
+    MSIDB2CAuthority *authority = [[self.class allocWithZone:zone] initWithURL:_url validateFormat:NO context:nil error:nil];
+    
+    authority.openIdConfigurationEndpoint = [_openIdConfigurationEndpoint copyWithZone:zone];
+    authority.metadata = self.metadata;
+    authority.url = [_url copyWithZone:zone];
```

Comment:
Does authority need to be set again? It's already passed in initializer.

Reply:
you mean URL? nope, I think we can skip this.

Reply:
Agreed and removed.